.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "ix-build-debug",
+      "command": "sh .vscode/ix-build-debug.sh",
+      "type": "shell",
+      "args": [],
+      "group": "test",
+    },
+    {
+      "label": "build-testrunner",
+      "command": "sh .vscode/build-testrunner.sh",
+      "type": "shell",
+      "args": [],
+      "group": {
+        "kind": "test",
+        "isDefault": true
+      },
+      "dependsOn": [
```

Comment:
Just to be clear, this makes it so this task will not until the defined task runs.